presto-main/src/test/java/com/facebook/presto/block/TestMapBlock.java

```diff
@@ -111,6 +113,15 @@ private void testWith(Map<String, Long>[] expectedValues)
         assertBlockFilteredPositions(expectedValues, block, () -> blockBuilder.newBlockBuilderLike(null), 0, 1, 3, 4, 7);
         assertBlockFilteredPositions(expectedValues, block, () -> blockBuilder.newBlockBuilderLike(null), 2, 3, 5, 6);
 
+        // Re-create a MapBlock without hashtables built to test getRegion.
+        Block blockForGetRegion = createBlockWithValuesFromKeyValueBlock(expectedValues);
+
+        int offset = (int) floor(blockForGetRegion.getPositionCount() / 2.0);
+        int length = (int) ceil(blockForGetRegion.getPositionCount() / 2.0);
```

Comment:
This can be done with integer math:

```java
int length = (blockForGetRegion.getPositionCount() + 1) / 2;
```

See, for example, FormatUtils.ceil:

```java
private static int ceil(int dividend, int divisor)
{
    return ((dividend + divisor) - 1) / divisor;
}
```

Reply:
I just felt it looked nicer this way. Anyways it was already removed.